executorlib/cache/subprocess_spawner.py

```
@@ -2,10 +2,13 @@
 import time
 from typing import Optional
 
+from executorlib.standalone.inputcheck import check_file_exists
+
 
 def execute_in_subprocess(
     command: list,
     task_dependent_lst: list = [],
```

Ruff

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)